### ISSUE TITLE
SOS/SPS REST Bindings

### ISSUE THREAD

**Issue by alexrobin** [opensensorhub/osh-core#99]
Implement new SOS/SPS REST bindings spec in parallel to existing SOS/SPS web services. This will provide a simpler API for many developers.

This will clearly ease the development of new language bindings.

**Comment by Sam-Bolling** [comment]
Good afternoon.  I am curious what 'new SOS/SPS REST bindings spec' is being referred to in this issue's description?

Respectfully, Sam.

**Comment by alexrobin** [comment]
Duplicate of #142
